src/riak_kv_ts_util.erl

```diff
@@ -80,19 +80,21 @@ build_sql_record(Command, SQL, Cover) ->
 %% Convert the proplist obtained from the QL parser
 build_sql_record_int(select, SQL, Cover) ->
     T = proplists:get_value(tables, SQL),
-    F = proplists:get_value(fields, SQL),
+    Fields = proplists:get_value(fields, SQL),
```

Comment:
If you are going to expand a variable from a letter to a word, do 'em all or don't do 'em

...the pedantic king of vertical alignment writes...